recipes/pytest-icdiff/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pytest-icdiff" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f874e668d3c8e590e49a33748ebfd0449cc5537a4919e87f5c3260ac1713504
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - icdiff
```

Comment:
Because this depends on a package that is in the same PR you'll need to re-start the CIs after the feedstock creation due to a race condition.

Reply:
Will do, thanks!

Reply:
[You beat me to it](https://circleci.com/gh/conda-forge/pytest-icdiff-feedstock/2) seems. 😅 Thanks!

Reply:
Yep. It was easier since I had all the tabs open already :smile: